PowerPointLabs/PowerPointLabs/Settings.StyleCop

```diff
@@ -274,22 +274,22 @@
         </Rule>
         <Rule Name="SystemUsingDirectivesMustBePlacedBeforeOtherUsingDirectives">
```

Comment:
Can just remove the rule? by default, it should be `enabled`?
only those disabled rules are put here